### PR TITLE
Update xml-format.md

### PR DESCRIPTION
**Pull request by Beeblerox**

### CODE REVIEW

**Comment by Beeblerox** [comment]
Don't merge it yet. It's WIP
And i must apologize for my bad English

**Comment by Beeblerox** [comment]
I think i've finished. You can review this PR now

**Review by ConfidantCommunications — APPROVED** [review]
Very helpful! I will make some minor language changes after merge. Thanks!

**Pull request merged**
